akka-docs/src/main/paradox/scala/additional/deploy.md

```diff
@@ -5,7 +5,16 @@ Akka can be used in different ways:
  * As a library: used as a regular JAR on the classpath and/or in a web app, to
 be put into `WEB-INF/lib`
  * As an application packaged with [sbt-native-packager](https://github.com/sbt/sbt-native-packager)
- * As an application packaged and deployed using [Lightbend ConductR](http://www.lightbend.com/products/conductr).
+ 
+## Lightbend ConductR
+ 
+An Akka application can also be packaged and deployed using [Lightbend ConductR](https://conductr.lightbend.com/docs/2.2.x/DevQuickStart).
+
+ConductR is a solution for managing Lightbend Reactive Platform applications across a cluster of machines.
+It is reactive from the ground up thus enabling operations to provide the resiliency required to unleash the full benefits 
+of the Lightbend Reactive Platform in their organization.
+
```

Comment:
Just copied and pasted the first paragraph from https://conductr.lightbend.com/docs/2.2.x/DevQuickStart